src/fetch.js

```diff
@@ -0,0 +1,31 @@
+const fetchAPI = {
+  handleFetchError: (response) => {
+    if(!response.ok) {
+      throw 'We are having difficulty getting the data right now!';
+    }
+    return response;
+  },
```

Comment:
This is the smartest thing I've ever seen. Holy cow. I really like this idea and this function. Imma have to google throw and response.ok